codingTest_김준우/체육복 다른 풀이.js

```diff
@@ -0,0 +1,31 @@
+function solution(n, lost, reserve) {
+  let total = [];
+  let answer = 0;
+  for (let i = 0; i < n; i++) {
+    total[i] = 1;
+  }
+  lost.forEach((data) => (total[data - 1] -= 1));
+  reserve.forEach((data) => (total[data - 1] += 1));
+  for (let i = 0; i < total.length; i++) {
+    if (total[i] === 2 && total[i - 1] === 0) {
+      total[i - 1]++;
+      total[i]--;
+    } else if (total[i] === 2 && total[i + 1] === 0) {
+      total[i]--;
+      total[i + 1]++;
+    }
+  }
+
+  for (let i = 0; i < total.length; i++) {
+    if (total[i] >= 1) {
+      answer++;
+    }
+  }
+  return answer;
+}
```

Comment:
```suggestion
  return allStudent.filter(c => c > 0).length;
```
filter 메소드를 이용해서 1 이상인 배열의 요소가 몇인지 바로 리턴해 줄 수 있어요